pipelines/zendesk/__init__.py

```diff
@@ -85,6 +87,14 @@ def zendesk_support(credentials: Union[ZendeskCredentialsEmailPass, ZendeskCrede
 
 
 # Zendesk Support resources
+@dlt.resource(primary_key="timestamp")
+def ticket_events(zendesk_client: Zenpy, timestamp = dlt.sources.incremental("timestamp", initial_value=1683054527)) -> Iterator[TDataItem]:
```

Comment:
Could you please set initial value to 0? so it starts from the beginning of all time

Reply:
changed! :)